prisma/migrations/20230710222340_init/migration.sql

```diff
@@ -0,0 +1,35 @@
+-- CreateTable
+CREATE TABLE "Developer" (
+    "id" SERIAL NOT NULL,
+    "fullName" TEXT NOT NULL,
+    "email" TEXT NOT NULL,
```

Comment:
Did you read Prisma? In Prisma, you don't need to write your code like this

Reply:
Was settle by prisma dev commands